requirements.txt

```diff
@@ -1,10 +1,10 @@
 openapi-spec-validator==0.2.9
-PyJWT==1.7.1
+PyJWT==2.0.1
```

Comment:
On `inmanta-core` this dependency is still at 1.7.1. Is this one excluded from version bumps?

Reply:
I think this is recent bump then

Reply:
This one is ignored: https://github.com/inmanta/inmanta-core/pull/2609. Can I resume the bumps?

Reply:
Lets wait for the test PR I linked. When we bump it, it should resume.